scalyr_agent/scalyr_client.py

```diff
@@ -1502,16 +1506,16 @@ def __init__(
         self.__has_non_optimal_fields = False
         self.__num_optimal_fields = 0
 
-    def __set_attributes(self, thread_id, attributes, log_line_attributes=False):
+    def __set_attributes(self, thread_id, attributes):
         """ Set the attributes and thread id of an Event.
 
         In the default case of `log_line_attributes` being False this will not add the attributes to the serialization
         base because those attributes should be included in the attributes of the log. If `log_line_attributes` is True
         they will be serialized but not set to `__attrs`, so any events based off this one will not inherit them.
         """
         self.__thread_id = thread_id
-        if self.__disable_logfile_addevents_format or not log_line_attributes:
-            self.__attrs = attributes
+        self.__attrs = attributes
```

Comment:
Relook at the comments for this method.. I think those are implementation details not needed to be covered here.